src/main/java/MainPage3.java

```diff
@@ -0,0 +1,52 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class MainPage3 {
+    private WebDriver driver;
+
+    public MainPage3 (WebDriver driver){
+
+        this.driver = driver;
+    }
+
+    private By newPaste = By.id("postform-text");
+    private By syntax = By.xpath("//span[@id='select2-postform-format-container']");
```

Comment:
it's better to give element names with element type at the end - e.g. syntaxDropdown, etc